src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java

```diff
@@ -110,8 +114,11 @@ class BtcReleaseClientTest {
 
     @BeforeEach
     void setup() {
-        params = RegTestParams.get();
-        bridgeConstants = Constants.regtest().bridgeConstants;
+        // ensure confirmation difference always passes
+        Block blockWithTxWaitingForSignatures = mock(Block.class);
+        when(blockWithTxWaitingForSignatures.getNumber()).thenReturn(0L); 
+        when(blockStore.getBlockByHash(any())).thenReturn(blockWithTxWaitingForSignatures);
```

Comment:
Do not use any. Instead, set this stub in the test with the specific value which you expect the method to be called with.

Reply:
Updated